addons/vehicles/CfgVehicles/Viper.inc

```diff
@@ -10,7 +8,8 @@ class SWS_V_MBT_M700_Viper_Base : I_MBT_03_cannon_F
     typicalCargo[] = { "OPTRE_UNSC_ODST_Soldier" };
     editorPreview = QPATHTOF(data\viper\editor_preview.jpg);
 
-    radarType = 2;
+    armor = 800;
```

Comment:
From what I can see, it looks like it already was set to this. I_MBT_03_Cannon_F has that value at least.

Reply:
The armor value or the radarType?

Reply:
Looks like you're right, though. Idk why but I think I got it in my head that its armor was 650... Anyways, I'm gonna bump it to 900